locations/hours.py

```diff
@@ -3,6 +3,8 @@
 from collections import defaultdict
 
 DAYS = ["Mo", "Tu", "We", "Th", "Fr", "Sa", "Su"]
+DAYS_3_LETTERS = ["Mon", "Tue", "Wed", "Thu", "Fri", "Sat", "Sun"]
+DAYS_3_LETTERS_SUNDAY = DAYS_3_LETTERS[-1:] + DAYS_3_LETTERS[:-1]
```

Comment:
`_SUNDAY` may be a bit clearer as `_SUN_TO_MON` or `_FROM_SUNDAY` or `_SUNDAY_START`?

Reply:
Renamed to `DAYS_3_LETTERS_FROM_SUNDAY` in https://github.com/alltheplaces/alltheplaces/pull/6170/commits/5e58dce224fd759b728f7725b628e3a08294c794